ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -389,6 +389,12 @@ function check_upgrade_recommend_when_upgrade_inprogress() {
         # So far, this is a TP feature in OCP 4.18 and OCP 4.19, so need to enable the gate 
         export OC_ENABLE_CMD_UPGRADE_RECOMMEND=true
     fi
+
+    # https://issues.redhat.com/browse/OCPBUGS-52504
+    # RECOMMEND and PRECHECK command shoudl be able to work together
+    if [[ "${TARGET_MINOR_VERSION}" -eq "19" ]] ; then
```

Comment:
What about minor versions greater than 19?

Reply:
About about `-gt 19` so that verions greater than 19 are included

Reply:
Yes, but I only afraid is once PRECHECK GAed with OCP 4.20, then we don't need to export those env variables.

Reply:
got it, thanks